code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -958,6 +1017,30 @@ Pass a positive integer as an argument to override a bot's default speed.
 		eject += "<BR>"
 	return eject
 
+/mob/living/simple_animal/bot/proc/insertpai(mob/user, obj/item/paicard/card)
+	if(paicard)
+		to_chat(user, "<span class='warning'>A [paicard] is already inserted!</span>")
```

Comment:
will this return the name of the pAI card? If so it can lead to some awkward sentences.

Reply:
I *think* items in square brackets use the name variable? I think it would translate to "A personal AI device is already inserted!"

Don't quote me on that though.

Reply:
In that case, consider this resolved